apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/ManifestsList/ManifestItem/index.tsx

```diff
@@ -69,6 +75,31 @@ export default function ManifestItem({
     [colors, manifest.branch],
   );
 
+  const label = (text: string, style: Omit<ItemStyle, "color">) => {
+    const { badgeColor, borderColor, backgroundColor } = style;
+    return (
+      <Text
+        fontSize="9px"
+        width="auto"
+        paddingX={2}
+        paddingY={1}
+        borderWidth={1}
+        borderRadius={3}
+        borderStyle={"solid"}
+        flexGrow={0}
+        flexShrink={0}
+        overflow={"hidden"}
+        textTransform="uppercase"
+        color={badgeColor}
+        borderColor={borderColor}
+        backgroundColor={backgroundColor}
+        fontWeight="semiBold"
+      >
+        {text}
+      </Text>
+    );
+  };
+
```

Comment:
Why is this a function inside another component while it can just be a component outside call with jsx properly ?